specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2017-08-31/managedClusters.json

```diff
@@ -116,7 +116,8 @@
         ],
         "operationId": "ManagedClusters_GetUpgradeProfile",
         "summary": "Gets upgrade profile for a managed cluster.",
-        "description": "Gets the details of the upgrade profile for a managed cluster with a specified resource group and name.",
+        "deprecated": true,
+        "description": "Use ManagedClusters_ListCredential instead.",
```

Comment:
I think you are deprecating the wrong operation.  I believe you should deprecate operationId `ManagedClusters_GetAccessProfiles`, right?

Reply:
D'oh, my mistake. I'll fix.